.github/workflows/broken-links.yml

```diff
@@ -0,0 +1,29 @@
+name: Using htmlproofer to report out all Broken Links
+
+on: [push]
+
+jobs:
+  prose:
```

Comment:
This needs to be changed to something like `check_broken_links` in order to be working properly.